File: python3_solution_set2/2498. Frog Jump II.py
class Solution:
    def maxJump(self, stones: List[int]) -> int:
        if len(stones) == 2:
            return stones[-1] - stones[0]
        ans = -1
        for x in range(len(stones)-2):
            ans = max(ans, stones[x+2] - stones[x])

        return ans

File: cpp_solutions/2498. Frog Jump II.cpp
class Solution {
public:
    int maxJump(vector<int>& stones) {
        if (stones.size() == 2){
            return stones[1] - stones[0];
        } else {
            int ans = -1;
            for (int x=0; x<stones.size()-2;++x){
                ans = max(ans, stones[x+2] - stones[x]);
            }
            return ans;
        }
        
    }
};
